src/lib/components/data-table/column-helpers.test.ts

```
@@ -0,0 +1,135 @@
+
+		it('should create cell function', () => {
+			const column = createActionsColumn<TestEntity>('user', '/users');
+
+			expect(typeof column.cell).toBe('function');
+		});
+
+		it('should default canEdit and canDelete to true', () => {
+			const column = createActionsColumn<TestEntity>('user', '/users');
+
+			// Create a mock row
+			const mockRow = {
```

ESLint

[error] 'mockRow' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)